application/controllers/UserGroupController.php

```diff
@@ -100,98 +100,96 @@ public function actionIndex()
-                // @todo: Move the zebra striping to view
-                if ($bgcc == "evenrow") {
-                    $bgcc = "oddrow";
-                } else {
-                    $bgcc = "evenrow";
-                }
-                $userloop[$row]["userid"] = $oUser->uid;
-
-                //  output users
-                $userloop[$row]["rowclass"] = $bgcc;
-                if (Permission::model()->hasGlobalPermission('usergroups', 'update') && $oUser->parent_id == Yii::app()->session['loginID']) {
-                    $userloop[$row]["displayactions"] = true;
-                } else {
-                    $userloop[$row]["displayactions"] = false;
-                }
+            //  output users
+            $userloop[$row]["rowclass"] = $bgcc;
+            if (Permission::model()->hasGlobalPermission('usergroups', 'update') && $oUser->parent_id == Yii::app()->session['loginID']) {
+                $userloop[$row]["displayactions"] = true;
+            } else {
+                $userloop[$row]["displayactions"] = false;
+            }
 
-                $userloop[$row]["username"] = $oUser->users_name;
-                $userloop[$row]["email"] = $oUser->email;
+            $userloop[$row]["username"] = $oUser->users_name;
+            $userloop[$row]["email"] = $oUser->email;
 
-                $row++;
-            }
-            $aData["userloop"] = $userloop;
+            $row++;
+        }
+        $aData["userloop"] = $userloop;
 
-            $aSearchCriteria = new CDbCriteria();
-            $aSearchCriteria->compare("ugid", $ugid);
-            if (!Permission::model()->hasGlobalPermission('superadmin', 'read')) {
-                $aSearchCriteria->compare("owner_id", Yii::app()->session['loginID']);
-            }
-            $aFilteredUserGroups = UserGroup::model()->count($aSearchCriteria);
-
-            if ($aFilteredUserGroups > 0) {
-                $aData["useradddialog"] = true;
-
-                $aUsers = User::model()->findAll(['join' => "LEFT JOIN (SELECT uid AS id FROM {{user_in_groups}} WHERE ugid = {$ugid}) AS b ON t.uid = b.id", 'condition' => "id IS NULL ORDER BY users_name"]);
-                $aNewUserListData = CHtml::listData($aUsers, 'uid', function ($user) {
-                    return \CHtml::encode($user->users_name) . " (" . \CHtml::encode($user->full_name) . ')';
-                });
-                // Remove group owner because an owner is automatically member of a group
-                unset($aNewUserListData[$aUserInGroupsResult->owner_id]);
-                $aData["addableUsers"] = array('-1' => gT("Please choose...")) + $aNewUserListData;
-                $aData["useraddurl"] = "";
-            }
+        $aSearchCriteria = new CDbCriteria();
+        $aSearchCriteria->compare("ugid", $ugid);
+        if (!Permission::model()->hasGlobalPermission('superadmin', 'read')) {
+            $aSearchCriteria->compare("owner_id", Yii::app()->session['loginID']);
```

Comment:
Need decision about https://bugs.limesurvey.org/view.php?id=18289

And review about usage of `usercontrolSameGroupPolicy`